### PR TITLE
Virtis level 2 ingestion fix

### PR DESCRIPTION
**Pull request by krlberry**
Add a check for level 3 data to virtis ingestion before trying to read trailer bytes that don't exist in level 2 data.

Camera model is still only supported for level 3 (calibrated) data.

## Related Issue
#2508 

## Motivation and Context
Camera model update to support level 3 data broke level 2 data ingestion. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/CMakeLists.txt [hunk 1]
```diff
@@ -40,8 +40,8 @@ set(PACKAGE_VERSION    ${VERSION})
 set(PACKAGE_STRING     "${PACKAGE_NAME} ${VERSION}")
 
 # Other release information
-set(VERSION_DATE              "2018-10-26")
-set(RELEASE_STAGE             "beta") # (alpha, beta, stable)
+set(VERSION_DATE              "2019-05-29")
+set(RELEASE_STAGE             "stable") # (alpha, beta, stable)
```
Comment (SgStapleton) [inline isis/CMakeLists.txt]: Hmm... I thought that this file had already been updated recently...? Was this a change you made specifically for this PR?

Comment (krlberry) [reply isis/CMakeLists.txt]: No, that was something that somehow came along for the ride with the merge of dev.

Comment (krlberry) [reply isis/CMakeLists.txt]: @SgStapleton Fixed.

**Review by SgStapleton — APPROVED** [review]
Fairly simple change and looks like this should fix the issue.

**Pull request merged**